### ISSUE TITLE
The application 'XXXX' could not be found in the tenant 'YYYY'.

### ISSUE THREAD

**Issue by vincentwolface** [MicrosoftDocs/powerapps-docs#1110]
Hello, I always get this same error when I edit the "Crisis Communication app" once it is imported.
I tried on 2 tenants - same error, just the Guid that is different. Any idea?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6836ad5a-2de5-6223-0514-62c83ac38d17
* Version Independent ID: 8cf9565e-4998-bce4-b319-fedb7e9d835b
* Content: [Crisis communication app - sample template - PowerApps](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/sample-crisis-communication-app#import-and-set-up-the-crisis-communication-app)
* Content Source: [powerapps-docs/maker/canvas-apps/sample-crisis-communication-app.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/maker/canvas-apps/sample-crisis-communication-app.md)
* Service: **powerapps**
* GitHub Login: @matthewbolanos
* Microsoft Alias: **mabolan**

**Comment by matthewbolanos** [comment]
Do you mind sharing a screenshot of your issue?

**Comment by vincentwolface** [comment]
Hello Matthew, thanks for your time.
I tried again and actually, it is working now, I can edit the application after a few minutes.
A few minutes ago, clicking on the _Edit_ app menu was not working -> same error message. 
It is like the application was imported (no error following the import) but actions on it like _Edit_ or _Settings_ (see screenshot I took) or even _Delete_ were not working.
![Error_after_clicking_on_app_settings_menu](https://user-images.githubusercontent.com/10480009/76122527-85d2ce00-5ff6-11ea-8b4f-ace068f0954d.png)
